src/chrome/content/rules/ServiceNow.com.xml

```diff
@@ -55,19 +54,16 @@
 	<!--	Direct rewrites:
 				-->
 	<target host="servicenow.com" />
+	<target host="www.servicenow.com" />
 	<target host="community.servicenow.com" />
 	<target host="developer.servicenow.com" />
+	<target host="docs.servicenow.com" />
 	<target host="express.servicenow.com" />
+	<target host="knowledge.servicenow.com" />
```

Comment:
Ah sorry, HTTPS Everywhere currently is rewriting to a different location, but you just fixed this.